config-templates/argus/versioned/argus-config-1.0.0.yaml

```diff
@@ -28,7 +28,7 @@ etcdDiscoveryToken: ""
 # The image respository of the [Argus](https://hub.docker.com/r/logicmonitor/argus) container.
 imageRepository: logicmonitor/argus
 # The image tag of the [Argus](https://hub.docker.com/r/logicmonitor/argus/tags) container.
-imageTag: v6
+imageTag: v5
```

Comment:
Why is this changed to v5 ? for argus 1.0.0, it should be v6 right ?

Reply:
Yes it should be v6, correcting this